src/main/java/io/appform/dropwizard/actors/actor/MessageMetaData.java

```diff
@@ -0,0 +1,13 @@
+package io.appform.dropwizard.actors.actor;
+
+import lombok.Data;
+
+@Data
+public final class MessageMetaData {
```

Comment:
Metadata is one word. Rename to `MessageMetadata`

Reply:
renamed.